src/Compilers/Core/Portable/InternalUtilities/SetsRequiredMembersAttribute.cs

```diff
@@ -5,7 +5,7 @@
 // Copied from:
 // https://github.com/dotnet/runtime/blob/fdd104ec5e1d0d2aa24a6723995a98d0124f724b/src/libraries/System.Private.CoreLib/src/System/Diagnostics/CodeAnalysis/SetsRequiredMembersAttribute.cs
 
-#if !NET6_0_OR_GREATER
```

Comment:
Would `#if NETSTANDARD` be better, so that it is not updated every time we bump version?